icu4c/source/runConfigureICU

```diff
@@ -256,16 +256,16 @@ case $platform in
         CXX=g++; export CXX
         RELEASE_CFLAGS='-O3'
         RELEASE_CXXFLAGS='-O3'
-        DEBUG_CFLAGS='-g'
-        DEBUG_CXXFLAGS='-g'
+        DEBUG_CFLAGS='-g -gdwarf-4'
```

Comment:
I have concern about this. Should we leave this flag out of runConfigureICU but instead passing in from the command line or yml file?

Reply:
I can try passing it from the command line to see if it works that way.